formula1-service/src/test/java/cz/muni/pa165/teamwhite/formula1/service/facade/UnitFacadeImplTest.java

```diff
@@ -0,0 +1,98 @@
+package cz.muni.pa165.teamwhite.formula1.service.facade;
+
+import cz.muni.pa165.teamwhite.formula1.dto.UserDTO;
+import cz.muni.pa165.teamwhite.formula1.enums.Role;
+import cz.muni.pa165.teamwhite.formula1.facade.UserFacade;
+import cz.muni.pa165.teamwhite.formula1.service.ServiceConfiguration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.transaction.Transactional;
+
+/**
+ * @author Karolina Hecova
+ */
+@ContextConfiguration(classes = ServiceConfiguration.class)
+@Transactional
+@TestExecutionListeners(TransactionalTestExecutionListener.class)
+public class UnitFacadeImplTest extends AbstractTestNGSpringContextTests {
```

Comment:
nema to byt User? :D

Reply:
To uz je prejmenovane, ne? :smile:

Reply:
melo by to byt 😄 ... ve vetvi to je dobre